### PR TITLE
Fix/vonk 4248 add dbchange

### PR DESCRIPTION
**Pull request by cknaap**

### CODE REVIEW

File: tools/firely-server-ingest.rst [hunk 1]
```diff
@@ -279,6 +279,14 @@ Your license already permits the usage of FSI if it contains ``http://fire.ly/vo
...
 -------------
 
+.. _fsi_releasenotes_2.0.0
+
+Release 2.0.0, January 26th, 2023
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* Upgraded to work with the database schemas for :ref:`Firely Server 5.0.0-beta1<vonk_releasenotes_5_0_0-beta1>`
+* 
+
```
Comment (alexzautke) [inline tools/firely-server-ingest.rst:289]: And it fixes the indexing for versioned references

File: releasenotes/releasenotes_v5.rst [hunk 2]
```diff
@@ -3,13 +3,24 @@
...
 
 Release 5.0.0-beta1, January 19th, 2023
 ------------------------------
 .. attention::
     This is a beta release of Firely Server 5.0.0. Although the core functionality remains fully intact, parts of the public API have been removed or heavily modified. Please consult the list under section 'Plugin and Facade' and check whether your implementation is affected by these changes.
 
+Database
+^^^^^^^^
+
+#. Because of feature 6 below, the database was updated for both **SQL Server** and **MongoDB**. Firely Server will usually perform the upgrade automatically. See for details :ref:`migrations`.
```
Comment (alexzautke) [inline releasenotes/releasenotes_v5.rst:16]: I would suggest to spell out what the feature is instead of saying „feature 6“

**Review by alexzautke — CHANGES_REQUESTED** [review]

File: tools/firely-server-ingest.rst (hunk 1 above)
Comment (cknaap) [reply tools/firely-server-ingest.rst:289]: Added

File: releasenotes/releasenotes_v5.rst (hunk 2 above)
Comment (cknaap) [reply releasenotes/releasenotes_v5.rst]: done

**Review by alexzautke — APPROVED** [review]

**Pull request merged**